Interface/GIGraph.m

```diff
@@ -313,19 +306,20 @@ - (void)_generateGraph {
   }
   
   // Make sure we have some tips left
-  if (tips.count == 0) {
+  if (tips.commits.count == 0) {
```

Comment:
You said above that `-commits` is expensive: why generate an entire array just to get the count?